File: Backend/twetter/src/main/java/com/bartdebever/twetter/services/ReplyService.java
package com.bartdebever.twetter.services;

import com.bartdebever.twetter.models.Reply;
import com.bartdebever.twetter.services.interfaces.IReplyService;

import java.util.List;

/**
 * Service to perform Crud actions for the Reply object.
 */
public class ReplyService extends CrudService<Reply> implements IReplyService {
    @Override
    public Reply getById(int id) {
        return getSession().get(Reply.class, id);
    }

    @Override
    public List<Reply> getAll() {
        return null;
    }
}

File: Backend/src/main/kotlin/services/ReplyService.kt
package services

import models.Reply
import services.interfaces.IReplyService
import javax.enterprise.context.RequestScoped

/**
 * Service to perform Crud actions for the Reply object.
 */
@RequestScoped
open class ReplyService : CrudService<Reply>(), IReplyService {

    override val all: List<Reply>? = null

    override fun getById(id: Int): Reply {
        return entityManager!!.find(Reply::class.java, id)
    }
}
